cli.md

```diff
@@ -44,12 +44,18 @@ If `--name` is not specified, CLI will look for an environment variable, e.g. `e
 ### Starting Projects
 * `t2 init` in the current directory, create a package.json and index.js with Hello World code. *Note that the index.js code doesn't yet work on Tessel 2*
 
+### Project Files
+Along with the package.json and index.js included in the `t2 init` process, there are some other files that may be useful for your project:
+* `.tesselignore` similar to .gitignore or [.npmignore](https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package), this file should list any files or directories you want ignored by the T2 bundling and deployment process. This is handy when using the `--full` flag, which tells T2 to bundle everything in the project directory.
+* `.tesselinclude` the overriding and opposite behavior of `.tesselignore`, this file should list any files or directories you want included with the T2 bundling and deployment process. This is handy for including non-JavaScript assets, like HTML, CSS, and images, for use within your project.
+
 ### Tessel Management
 * `t2 provision` authorize your computer to access a Tessel over SSH (USB-connected Tessel only)
 * `t2 list` show what Tessels are available over WiFi and USB.
 * `t2 rename` change the name of a Tessel
 
 ### Code Deployment 
+During code deployment, CLI looks for those `.tesselignore` and `.tesselinclude` files to let it know which files it should bundle up and push over to Tessel. In the default bundling process, CLI takes the file passed into the `run` or `push` commands and finds all its dependencies by following the 'require' statements, similar to how [Browserify](https://github.com/substack/browserify-handbook#how-browserify-works) works because that's what CLI uses under the hood.
```

Comment:
nitpicky, but weird to say "those" in the first line. Would just cut the word.
Last bit is a little awkward too, consider "statements, similar to how [Browserify](https://github.com/substack/browserify-handbook#how-browserify-works) works because that's what CLI uses under the hood." > "statements (we use [Browserify](https://github.com/substack/browserify-handbook#how-browserify-works) to do this)."